Assets/Scripts/Environment/InteractableObject.cs

```diff
@@ -5,12 +5,7 @@ public class InteractableObject : MonoBehaviour
     // Set this OnEnter and OnExit to avoiding repeatedly comparing tags
     public bool IsPlayerColliding { get; protected set; }
 
-    [SerializeField] new protected Collider collider;
-
-    protected void SetPlayerIsColliding(Collider other)
-    {
-        IsPlayerColliding = other.CompareTag(PlayerConstants.PlayerTag);
-    }

```

Comment:
I removed this function because if anything enters the collider that is not the player, it will set IsPlayerColliding even though the player is still within the collider.

Reply:
Cool, as long as it doesn't break anything else that's fine.